README.md

```diff
@@ -298,9 +298,11 @@ Errors like `[S_API FAIL] Tried to access Steam interface SteamUser021 before St
 ### Im using Apple silicon type of hardware, can I run this?
 You can try to insert in your docker-compose file this parameter `platform: linux/amd64` at the palworld service. This isnt a special fix for Apple silicon, but to run on other than x86 hosts. The support for arm exists only by enforcing x86 emulation, if that isnt to host already. Rosetta is doing the translation/emulation.
 
-### I have changed the settings, but some changes are un-affected in the server?
+### I have changed the `BaseCampWorkerMaxNum` settings, why are changes un-affected on the server?
 
-It is a known bug, for some setting options like `BaseCampWorkerMaxNum`, changing `PalWorldSettings.ini` will not affect the server. Here is a workaround: https://github.com/legoduded/palworld-worldoptions
+It is a confirm bug, that changing `BaseCampWorkerMaxNum` in the `PalWorldSettings.ini` has no affect on the server. There are tools out there to help with this, like this one: <https://github.com/legoduded/palworld-worldoption>
+
+Please note that adding `WorldOption.sav` will breaking `PalWorldSetting.ini` and if that happens all the other settings won't be changeable anymore unless that specific tool is used everytime.
```

Comment:
Add Bold here just to enphasize the warning, after that ok to go

Reply:
on 305 line

Reply:
```suggestion
**PLEASE NOTE** that adding `WorldOption.sav` will breaking `PalWorldSetting.ini` and if that happens all the other settings won't be changeable anymore unless that specific tool is used everytime.
```